package.json

```diff
@@ -17,28 +17,31 @@
         "url": "https://github.com/aws-observability/aws-rum-web.git"
     },
     "scripts": {
-        "server": "npm run build:schemas && webpack-dev-server --config webpack/webpack.dev.js",
-        "test": "npm run build:webpack:dev && jest -c jest.unit.config.js",
-        "release": "npm run build:webpack:prod && npm run build:es && npm run build:cjs",
+        "server": "webpack-dev-server --config webpack/webpack.dev.js",
+        "start": "npm run server",
+        "pretest": "npm run build:schemas",
```

Comment:
In the original, we run `build:webpack:dev` then run the tests. Here, it seems that we run `build:schemas` instead; is this intentional? Could you explain the change?

Reply:
We don't need to build we pack before running the tests, but the schemas need to be there